PolloPollo.Repository/ApplicationRepository.cs

```diff
@@ -117,6 +117,28 @@ public async Task<ApplicationDTO> FindAsync(int applicationId)
             return application;
         }
 
+        /// <summary>
+        /// Update state of application
+        /// </summary>
+        /// <param name="dto"></param>
+        /// <returns></returns>
+        public async Task<bool> UpdateAsync(ApplicationUpdateDTO dto)
+        {
+            var application = await _context.Applications.
```

Comment:
Should probably have the same checks as for the deletion, so only the correct user can update the status.